src/components/30-organisms/footer/index.scss

```diff
@@ -111,20 +118,20 @@
     display: flex;
     justify-content: space-between;
     color: $color-prim-white;
-    background-color: $color-dark-indigo;
-    padding: 20px 30px 20px 30px;
+    background-color: $color-ocean-blue;
+    padding: 20px 30px;
     width: 100%;
     text-align: left;
 
     border: none;
     outline: 0;
 
-    border-bottom: 1px solid $color-azure;
+    border-bottom: 1px solid rgba($color-prim-white, 0.25);
     text-transform: uppercase;
 
     svg {
-      width: 10px;
```

Comment:
Why does it need this change? It's not part of the story.

Reply:
I was testing something and accidentally pushed it, I will remove it